PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.hdinsight/src/com/microsoft/azuretools/hdinsight/projects/CreateProjectUtil.java

```diff
@@ -98,7 +98,14 @@ public static void createSampleFile(@NotNull String id, @NotNull String projectN
 			AppInsightsClient.create(Messages.SparkProjectSystemScalaSampleCreation, null);
 			copyFileTo(Scala_Cluster_Run_Sample, rootPath);
 			break;
+		case "com.microsoft.azure.hdinsight.scala.projwizard":
+			AppInsightsClient.create(Messages.SparkProjectSystemScalaCreation, null);
```

Comment:
Yes

---
In reply to: [120295355](https://github.com/Microsoft/azure-tools-for-java/pull/595#discussion_r120295355) [](ancestors = 120295355)